crates/utils/src/utils/validation.rs

```diff
@@ -150,7 +147,8 @@ pub fn is_valid_matrix_id(matrix_id: &str) -> LemmyResult<()> {
 }
 
 pub fn is_valid_post_title(title: &str) -> LemmyResult<()> {
-  let check = VALID_POST_TITLE_REGEX.is_match(title) && !has_newline(title);
+  let length = title.trim().len();
+  let check = length >= 3 && length <= 200 && !has_newline(title);
```

Comment:
You can also do this with [range contains](https://stackoverflow.com/questions/28265036/how-can-i-test-if-a-value-lies-within-a-range) but I leave that up to you.

Reply:
Clippy beat me to it :laughing: 

https://woodpecker.join-lemmy.org/repos/129/pipeline/4064/17

Reply:
Thank you. I'm learning Rust by contributing and your feedback helps.

Reply:
No probs!